src/scss/components/_inputs.scss

```diff
@@ -144,6 +144,29 @@ $input-select-margin-right: 1;
   }
 }
 
+.usa-checkbox__input--tile + .usa-checkbox__label--illustrated,
+.usa-radio__input--tile + .usa-radio__label--illustrated {
+  @include at-media('tablet') {
+    display: flex;
+  }
+
+  .usa-radio__image,
+  .usa-checkbox__image {
+    width: 1rem;
+    float: left;
```

Comment:
Does this need to be floated if we're also using flexbox?

Reply:
This needs to be floated because on mobile, we are not using flexbox. On mobile, the design has the text wrapped around the image. 
![image](https://user-images.githubusercontent.com/17969963/154295878-c6ac223e-bb1b-41a6-b65b-d8eac407897c.png)

Reply:
Ah, good point, I missed that! Though, that raises another question: If the heading spans multiple lines, should it wrap under the icon (float behavior), or should the title text always be aligned with itself?

![image](https://user-images.githubusercontent.com/1779930/154296466-97e6d9d1-cb9d-4590-8799-629bd616e542.png)

Reply:
yes, this is/was a problem. I took the image out of the container and applied the styles to the image itself. This is now what we are working with
![image](https://user-images.githubusercontent.com/17969963/154320186-464d3a95-3a0d-4277-9310-38675a32fb86.png)